XCUITests/TopTabsTest.swift

```diff
@@ -63,7 +63,7 @@ class TopTabsTest: BaseTestCase {
         waitforExistence(app.webViews.links.staticTexts["More information..."])
         app.webViews.links.staticTexts["More information..."].press(forDuration: 5)
         app.buttons["Open in New Tab"].tap()
-        app.buttons["Switch"].tap()
+        sleep(2)
```

Comment:
How about if you do a waitForExistence call on Switch button? (waitforExistence(app.buttons["Switch"])) I think this reacts a bit faster, since it doesn't wait until the screen is idle.
OR, alternatively, you don't need sleep(2) at all, since it'll wait until the screen becomes idle, then try to tap UI to go to Taptray.